src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/AccessResult.java

```diff
@@ -27,8 +27,12 @@ enum AccessResult {
 
     /** Represents that access is allowed. */
     ALLOWED,
-    /** Represents that access is disallowed. */
+    /**
+     * Represents that access is disallowed.
+     */
     DISALLOWED,
+    /** Represents that access is forbidden. */
+    FORBIDDEN,
```

Comment:
we should have comment that they are synonyms.

Reply:
@apiNote synonym for {@link #DISALLOWED}.